openpype/modules/ftrack/plugins/publish/integrate_ftrack_instances.py

```diff
@@ -143,70 +145,87 @@ def process(self, instance):
         unmanaged_location_name = "ftrack.unmanaged"
         ftrack_server_location_name = "ftrack.server"
 
+        # check if any outputName keys are in review_representations
+        # also check if any outputName keys are in thumbnail_representations
+        synced_multiple_output_names = []
```

Comment:
I'm maybe blind, but the variable is not used anywhere.

Reply:
fixed here e6c050403871e45ccc7c0423ac107e2b8d9a8b02